receiver/httpcheckreceiver/config_test.go

```diff
@@ -41,8 +49,12 @@ func TestValidate(t *testing.T) {
 		{
 			desc: "valid config",
 			cfg: &Config{
-				HTTPClientSettings: confighttp.HTTPClientSettings{
-					Endpoint: "https://opentelemetry.io",
+				Targets: []*targetConfig{
```

Comment:
Worth adding a test with multiple endpoints? Just to confirm that any one invalid endpoint raises the expected error

Reply:
Added new test with multiple target with one invalid endpoint above.

Reply:
Thanks!